website/docs/r/ess_alarm.html.markdown

```diff
@@ -84,6 +84,7 @@ The following arguments are supported:
 
 * `name` - (Optional) The name for ess alarm.
 * `description` - (Optional) The description for the alarm.
+* `enable` - (Optional, Available in 1.48.0+) If enable specific ess alarm. Default to true.
```

Comment:
`If enable specific ess alarm. Default to true.` should be `Whether to enable the specified ESS alarm. Default to true.`

Reply:
fixed